cloudamqp/resource_cloudamqp_integration_metric_prometheus_test.go

```diff
@@ -329,3 +329,129 @@ func TestAccIntegrationMetricPrometheusAzureMonitor_Update(t *testing.T) {
 		},
 	})
 }
+
+// TestAccIntegrationMetricPrometheusSplunkV2_Basic: Add Splunk v2 prometheus metric integration and import.
+func TestAccIntegrationMetricPrometheusSplunkV2_Basic(t *testing.T) {
+	var (
+		fileNames                   = []string{"instance", "integrations/metrics/integration_metric_prometheus_splunk_v2"}
+		instanceResourceName        = "cloudamqp_instance.instance"
+		prometheusSplunkResourceName = "cloudamqp_integration_metric_prometheus.splunk_v2"
+
+		params = map[string]string{
+			"InstanceName":   "TestAccIntegrationMetricPrometheusSplunkV2_Basic",
+			"InstanceID":     fmt.Sprintf("%s.id", instanceResourceName),
+			"InstancePlan":   "bunny-1",
+			"SplunkToken":    "12345678-1234-1234-1234-123456789012",
```

Comment:
```suggestion
			"SplunkToken":   os.Getenv("SPLUNK_TOKEN") ,
```

Related to all keys, even if they are dummy keys. You can fetch them from a .env file (I use a different one for the integration tests). [Sanitizer](https://github.com/cloudamqp/terraform-provider-cloudamqp/blob/main/cloudamqp/provider_test.go#L213) will pick it up and change the key in the recording and switch it with `SPLUNK_TOKEN`.

In order to be able to record/playback the VCR fixture I had to add `ExpectNonEmptyPlan: true,` in each test step. [Example](https://github.com/cloudamqp/terraform-provider-cloudamqp/blob/main/cloudamqp/resource_cloudamqp_integration_metric_test.go#L13-L51)

To avoid re-record all tests, should be enough to find and replace the keys in each body.
Find `12345678-1234-1234-1234-123456789012`, replace with `SPLUNK_TOKEN`.

Result will be similar to

```json
{"access_key_id":"CLOUDWATCH_ACCESS_KEY_ID","region":"us-east-1","secret_access_key":"CLOUDWATCH_SECRET_ACCESS_KEY","tags":"env=test,region=us-east-1"}
```